cmd/skopeo/inspect.go

```diff
@@ -106,8 +106,9 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 		}
 	}()
 
+	unparsedInstance := image.UnparsedInstance(src, nil)
 	if err := retry.IfNecessary(ctx, func() error {
-		rawManifest, _, err = src.GetManifest(ctx, nil)
```

Comment:
I have a dim memory we talked about this before but is there a reason why `GetManifest` does not verify by default? We're just talking about some small JSON so the overhead of sha256 on it has to be pretty small.

Reply:
There are ~11 implementations in c/image + several outside, added over time; trusting that all of them are going to do the right thing is hard to enforce.

The idea originally was that the transports only deal with “I/O” and centralized code in c/image, primarily c/image/copy, provides a _single_ location where the enforcement provably happens.

That doesn’t really work with the various external ImageSource users… compare https://github.com/containers/image/pull/2742 .

We might provide some other interface, but we can’t prevent direct calls to `GetManifest` without breaking the stable API.

Reply:
> The idea originally was that the transports only deal with “I/O” and centralized code in c/image, primarily c/image/copy, provides a single location where the enforcement provably happens.

Maybe the way to think of this is that we should offer a "copy metadata to *memory*" operation?